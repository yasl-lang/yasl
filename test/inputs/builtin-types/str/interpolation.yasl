let $x = 10
let $y = 12
echo "$x is #{$x}, #{$y}."
echo "$x is #{$x        }, #{$y}"
echo "$x$y is #{$x  }#{$y}  "->tostr('r')

echo "hex: #{$x:x}, dec: #{$x:d}, bin: #{$x:b}"
