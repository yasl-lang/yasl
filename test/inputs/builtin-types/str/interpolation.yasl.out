$x is 10, 12.
$x is 10, 12
'$x$y is 1012  '
hex: 0xa, dec: 10, bin: 0b1010
